redeem-bot/library/tezos.js

```diff
@@ -73,14 +73,29 @@ module.exports = class Tezos {
    * @param address tezos address for the account
    */
   async tokenBalance(tokenContract, address) {
-    const key = TezosMessageUtils.encodeBigMapKey(
+    let key = TezosMessageUtils.encodeBigMapKey(
```

Comment:
npm supports github includes, perhaps reuse /library by making it a separate repo